src/main/scala/devices/uart/UARTTx.scala

```diff
@@ -29,11 +29,12 @@ class UARTTx(c: UARTParams) extends Module {
   io.out := out
 
   val plusarg_tx = PlusArg("uart_tx", 1, "Enable/disable the TX to speed up simulation").orR
+  val plusarg_printf = PlusArg("uart_tx_printf", 0, "Enable/disable the TX").orR
```

Comment:
Should this default to 1? Otherwise the behavior will change for end users.

Reply:
Ok